### ISSUE TITLE
Download area

### ISSUE THREAD

**Issue by Aram4ik** [agilecube/TGS#4]
Is missing. We talked that it should be possible to add download files. Just 2 inputs - name and url. We will upload files through FTP

**Comment by Aram4ik** [comment]
I see it in user area but don't see the place where from admin can add files to user's view

**Comment by agilecubeteam** [comment]
As per discussion, Following is requirement. That requirement i have suggested, but you said it will be just static page.  See your reply marked in following image. 

![issue-4](https://cloud.githubusercontent.com/assets/14108944/10072091/83fc5540-62dc-11e5-9991-5437a677aed1.png)

**Comment by Aram4ik** [comment]
Yes, but where I have to upload files to be appeared in download list?

**Comment by agilecube** [comment]
You can give all files as of now. 

Later on we will give you instruction note on how to update this page when added more files.

**Comment by Aram4ik** [comment]
When we transfer all to our hostings right? If so then wi will upload all files directly there, no need to do it twice

**Comment by agilecube** [comment]
Sure, putting on Hold for now. We shall do this as soon as TGS are on live domain.
